src/Views/ProgressView.vala

```diff
@@ -398,20 +265,20 @@ public class ProgressView : AbstractInstallerView {
         }
 
         foreach (Installer.Mount m in lvm_devices) {
-            var vg = m.parent_disk.offset (12);
+            var vg = m.parent_disk.offset (12).replace ("--", "-");
```

Comment:
Why is this needed?

Reply:
/dev/mapper has special rules for handling the `-` character in the device path, because the `-` character is used to separate the volume group identifier from the logical volume identifier. Someone had the installer fail because they were trying to install to a VG with `-` in the vg identifier.

Reply:
It might be worth adding a comment for that, then; otherwise this is spooky magic at a distance